web/search.dart

```diff
@@ -334,7 +334,16 @@ void initializeSearch(
       var noResults = document.createElement('div')
         ..classes.add('search-summary')
         ..innerHtml =
-            'There was not a match for "$input". Please try another search.';
+            'There was not a match for "$input". Want to try searching from additional Dart-related sites? ';
+
+      var buildLink = Uri.parse(
+              'https://dart.dev/search?cx=011220921317074318178%3A_yy-tmb5t_i&ie=UTF-8&hl=en&q=')
+          .replace(queryParameters: {'q': input});
+      var link = document.createElement('a')
+        ..setAttribute('href', buildLink.toString())
+        ..classes.add('seach-options')
+        ..innerHtml = 'click here to search in dart.dev.';
```

Comment:
Maybe something more concise and focused on the action (we try [avoid "click here"](https://developers.google.com/style/link-text?hl=en#write-link-text)).:

```suggestion
        ..innerHtml = 'Search on dart.dev.';
```

Reply:
Also, I'm not super familiar, but could we use [`text`](https://api.dart.dev/stable/dart-html/Node/text.html) here to avoid the browser unnecessarily parsing the specified string?

```suggestion
        ..text = 'click here to search in dart.dev.';
```

Reply:
Addressed in new commit